examples/todo/imports/ui/profile/ProfileContainer.js

```diff
@@ -19,11 +21,58 @@ export const editProfileMutation = graphql(
   }
 );
 
+
 const loggedUser = getLoggedUserContext();
 
+handleSubmitProfile = (editProfileState) => !!editProfileState.firstName.trim()
```

Comment:
is it a validation method, right? the name is wrong then.

Reply:
I think so.